website/models.py

```diff
@@ -582,3 +584,31 @@ class Monitor(models.Model):
 
     def __str__(self):
         return f"Monitor for {self.url} by {self.user}"
+
+
+class Bid(models.Model):
+    issue_url = models.URLField()
+    user = models.CharField(default="Add user", max_length=30, null=True, blank=True)
+    created = models.DateTimeField(default=timezone.now)
+    modified = models.DateTimeField(default=timezone.now)
```

Comment:
```suggestion
    created_at = models.DateTimeField(default=timezone.now)
    modified_at = models.DateTimeField(default=timezone.now)
```

Also take a look at https://docs.djangoproject.com/en/5.0/ref/models/fields/#datefield -- `auto_now` and `auto_now_add`